### PR TITLE
[WIP] I ssue/publising platform

### PR DESCRIPTION
**Pull request by SaadAli-rolustech**
**What does this PR?**
This PR consists of the functionality from which app, the content was published for each post.

**Where should the reviewer start?**
File name: postDisplayView.tsx

### Steps to reproduce

**Issue number**
https://github.com/ecency/ecency-mobile/issues/2670

 **Screenshots/Video**
Screenshot 2023-04-19 at 4 43 07 PM

### CODE REVIEW

File: src/config/ecencyApi.ts [hunk 1]
```diff
@@ -2,13 +2,15 @@ import axios from 'axios';
 import Config from 'react-native-config';
 import VersionNumber from 'react-native-version-number';
 import { get } from 'lodash';
+import Reactotron from 'reactotron-react-native';
```
Comment (noumantahir) [inline src/config/ecencyApi.ts:5]: We do no use `Reactotron` imports directly in production code, as production build strips reactotron-react-native package and cause app to crash if not in debug mode... there is a workaround to it by using `console.tron` but I have not configured it in this project yet. we usually use regular` console.log`, `console.debug` for now as it is configured to be removed before production.

File: ios/Ecency.xcodeproj/project.pbxproj
Comment (noumantahir) [inline ios/Ecency.xcodeproj/project.pbxproj:1]: try not to include both android and ios configs files in PR, unless some package exclusively updated or configuration is deliberately change. both of which does not seem to be a case here... usually config changes that slip to our CI/CD often messes with the production builds

File: src/components/postView/view/postDisplayView.tsx [hunk 2]
```diff
@@ -284,6 +287,9 @@ const PostDisplayView = ({
                   <Text style={styles.footerName}>{` ${author || post.author} `}</Text>
                   {formatedTime}
                 </Text>
+                <Text style={styles.publishedAppName}>
+                  Published using{` ${capitalize((post?.json_metadata?.app).split('/')[0])} `}
```
Comment (noumantahir) [inline src/components/postView/view/postDisplayView.tsx:291]: updated presentation as suggested by Feruz here... https://discord.com/channels/@me/920267778190086205/1098233745959555112

**Review by noumantahir — CHANGES_REQUESTED** [review]
@SaadAli-rolustech  good effort overall, just a few simple things needs addressing....

File: ios/Ecency.xcodeproj/project.pbxproj
Comment (SaadAli-rolustech) [reply ios/Ecency.xcodeproj/project.pbxproj:1]: Noted.

File: src/components/postView/view/postDisplayView.tsx [hunk 3]
```diff
@@ -282,6 +284,7 @@ const PostDisplayView = ({
                 <Text style={styles.footerText}>
                   Posted by
```
Comment (noumantahir) [inline src/components/postView/view/postDisplayView.tsx:285]: as suggested by feruz, use translations here... there is format supported by intl that supports injecting values in translation entries....

Code: `intl.formatMessage({id:'some_id'}, {username:'some_value_1', appName:'some_value_2'})`
en_US entry:` "some_id":"Posted by $usename via $appName"`

File: src/utils/wallet.ts [hunk 4]
```diff
@@ -515,42 +528,47 @@ const fetchEngineTokensData = async (username: string, hivePrice: number, vsCurr
...
     }
   } catch (err) {
-    console.warn("failed to get engine tokens data", err);
+    console.warn('failed to get engine tokens data', err);
   }
 
   return engineCoinData;
+};
 
-
```
Comment (noumantahir) [inline src/utils/wallet.ts:538]: try not to include all lint results in PR, I do have plans to an app wide some time soon, the ts rules needs to be updated before we can properly do that... for now only hand pick linted files that you change in a particular PR. Another tip regarding lint.

I personally usually skip pre-commit hooks and lint manually before PR as some times lint can also break the app, not doing auto commit with lint gives me a chance to make sure lint did not break anything in app

**Review by noumantahir — CHANGES_REQUESTED** [review]
Just some small change requests. it might seem a little daunting getting yourself familiar with how we work, but ones you get accustom to the process it should smooth out and feel free to drop suggestions to improving and automating the process as well, always open to discuss and adapt.

**Pull request closed without merging**